error_translator.go

```diff
@@ -5,14 +5,16 @@ import (
 	"gorm.io/gorm"
 )
 
-var errCodes = map[string]string{
-	"uniqueConstraint": "23505",
+var pgerrodesToGormErrors = map[string]error{
```

Comment:
If you see here https://github.com/go-gorm/postgres/pull/179, these errors are not just for pg anymore, maybe we can rename this to `errDesToGormErrs`?